Splunk/ceis/dashboards/ceis_executive_summary.xml

```diff
@@ -0,0 +1,590 @@
+        <option name="rangeColors">["0x53a051","0x0877a6","0xf8be34","0xf1813f","0xdc4e41"]</option>
+        <option name="refresh.display">progressbar</option>
+        <option name="showTrendIndicator">1</option>
+        <option name="trendColorInterpretation">inverse</option>
+        <option name="trendDisplayMode">absolute</option>
+        <option name="trendInterval">-5m</option>
+        <option name="underLabel">Slow</option>
+        <option name="unit">%</option>
+        <option name="unitPosition">after</option>
+        <option name="useColors">1</option>
+      </single>
+    </panel>
+  </row>
+  <row>
+    <panel>
+      <single>
+        <search>
+          <query>
+| makeresults 
+| eval title="Daily" 
+| eval colorCode=1
+| stats values(colorCode) as colorCode by title
+| eval range=if(isnotnull(title),"guarded","low")
+          </query>
+          <earliest>-1m@m</earliest>
+          <latest>now</latest>
+        </search>
+        <option name="colorBy">value</option>
+        <option name="colorMode">block</option>
+        <option name="drilldown">none</option>
+        <option name="field">title</option>
+        <option name="height">60</option>
+        <option name="rangeColors">["0x325b99","0x3c444d","0x3c444d","0x3c444d","0x3c444d"]</option>
+        <option name="useColors">0</option>
+      </single>
+    </panel>
+    <panel>
+      <single>
+        <search>
+          <query>
+| makeresults 
+| eval title="Weekly" 
+| eval colorCode=1
+| stats values(colorCode) as colorCode by title
+| eval range=if(isnotnull(title),"guarded","low")
+          </query>
+          <earliest>-1m@m</earliest>
+          <latest>now</latest>
+        </search>
+        <option name="colorBy">value</option>
+        <option name="colorMode">block</option>
+        <option name="drilldown">none</option>
+        <option name="field">title</option>
+        <option name="height">60</option>
+        <option name="rangeColors">["0x325b99","0x3c444d","0x3c444d","0x3c444d","0x3c444d"]</option>
+        <option name="useColors">0</option>
+      </single>
+    </panel>
+    <panel>
+      <single>
+        <search>
+          <query>
+| makeresults 
+| eval title="Monthly" 
+| eval colorCode=1
+| stats values(colorCode) as colorCode by title
+| eval range=if(isnotnull(title),"guarded","low")
+          </query>
+          <earliest>-1m@m</earliest>
+          <latest>now</latest>
+        </search>
+        <option name="colorBy">value</option>
+        <option name="colorMode">block</option>
+        <option name="drilldown">none</option>
+        <option name="field">title</option>
+        <option name="height">60</option>
+        <option name="rangeColors">["0x325b99","0x3c444d","0x3c444d","0x3c444d","0x3c444d"]</option>
+        <option name="useColors">0</option>
+      </single>
+    </panel>
+  </row>
+  <row>
+    <panel>
+      <title>Traffic reliability</title>
+      <chart>
+        <search base="webPerformance_1d">
```

Comment:
Can we use the same search base for all panels?  Also, I found CSV isn't as efficient as it should be with this dataset.  We should do a saved search, make it public and reference that in the dashboard.